pkg/status/status.go

```diff
@@ -75,6 +85,21 @@ func Get(ctx context.Context, opts *Options) (status *Status, ok bool, err error
 		}
 	}
 
+	if stringInSlice("registry", opts.Services) {
+		registryCfg := cfg.Services.ServiceRegistry
+		if registryCfg != nil && registryCfg.InstanceID != "" {
+			// nolint:govet
+			registry, newErr := getRegistryStatus(ctx, api.ServiceRegistryMgmt(), registryCfg.InstanceID)
+			if newErr != nil {
+				return status, ok, err
+			}
+			status.Registry = registry
+			ok = true
+		} else {
+			logger.Debug("No service registry is currently used, skipping status check")
```

Comment:
That is going to be printed when dev mode is disabled. Since this is debug message I think we can leave with it. 
Alternatively entire service registry part should be wrapped in dev check (preferred)

Reply:
Enclosing all of it inside a condition checking dev preview.